pkg/operator/render.go

```diff
@@ -67,19 +69,7 @@ func createDiscoveredControllerConfigSpec(infra *configv1.Infrastructure, networ
 		return nil, err
 	}
 
-	platform := "none"
-	switch infra.Status.Platform {
-	case configv1.AWSPlatformType:
-		platform = "aws"
-	case configv1.AzurePlatformType:
-		platform = "azure"
-	case configv1.OpenStackPlatformType:
-		platform = "openstack"
-	case configv1.LibvirtPlatformType:
-		platform = "libvirt"
-	case configv1.VSpherePlatformType:
-		platform = "vsphere"
-	}
+	platform := strings.ToLower(string(infra.Status.PlatformStatus.Type))
```

Comment:
the only semantic difference is that we don't fallback to a `none` platform right? cc @abhinavdahiya for confirmation

Reply:
actually looks like the none platform is defined? @runcom not sure if that's used by installer though.
https://github.com/openshift/api/blob/master/config/v1/types_infrastructure.go

Reply:
oh great, nevermind this comment then

Reply:
question: is there any way the platform won't be set to anything (not None, but empty)?  maybe we should check to make sure it has a value?